src/ucp/stream/stream_multi.c

```diff
@@ -18,6 +18,13 @@
 /* Convenience macros for setting eager protocols descriptions  */
 #define UCP_PROTO_STREAM_BCOPY_DESC \
     "stream " UCP_PROTO_COPY_IN_DESC " " UCP_PROTO_COPY_OUT_DESC
+#define UCP_PROTO_STREAM_ZCOPY_DESC \
+    "stream " UCP_PROTO_ZCOPY_DESC " " UCP_PROTO_COPY_OUT_DESC
+
+static int ucp_stream_check_op(const ucp_proto_init_params_t *init_params)
+{
+    return ucp_proto_init_check_op(init_params, UCS_BIT(UCP_OP_ID_STREAM_SEND));
+}
```

Comment:
Sorry, didn't noticed that `ucp_proto_multi_init` is also the same in both initialize methods. 
```suggestion
static ucs_status_t
ucp_stream_multi_init(const ucp_proto_multi_init_params_t *params)
{
    if (!ucp_proto_init_check_op(&params->super.super,
                                 UCS_BIT(UCP_OP_ID_STREAM_SEND))) {
        return UCS_ERR_UNSUPPORTED;
    }

    return ucp_proto_multi_init(params, params->super.super.priv,
                                params->super.super.priv_size);
}
```

, and 
```C
static ucs_status_t
ucp_stream_multi_bcopy_init(const ucp_proto_init_params_t *init_params)
{
    ...
    return ucp_stream_multi_init(&params);
}
...
static ucs_status_t
ucp_stream_multi_zcopy_init(const ucp_proto_init_params_t *init_params)
{
    ...
    return ucp_stream_multi_init(&params);
}
```